src/runtimes/docker/utils/normalizeContainerOS.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Return a normalized string for the OS name specifically for Docker
+ * @param os pre-normalized OS name
+ * @returns normalized OS name
+ */
+export function normalizeContainerOS(os?: string): ContainerOS {
+    switch (os?.toLocaleLowerCase() || 'linux') {
+        case 'windows':
+            return 'windows';
+        case 'mac':
+            return 'darwin';
+        case 'linux':
```

Comment:
I'd argue for returning "linux" if os is "linux" or null/undefined, but actually returning the value of os otherwise.

Reply:
Agreed. We should also cover empty string if needed.

Reply:
agreed